### PR TITLE
Bug 1147324 - Showing STK fullscreen when no statusbar r=kgrandon

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=839e9cdb10dbc6bea61ba49d13939679ab21e4bb

File: apps/system/js/icc.js [hunk 1]
```diff
@@ -329,6 +329,12 @@ var icc = {
   },
 
   resize: function() {
+    if (!StatusBar.height) {
```
Comment (KevinGrandon) [inline apps/system/js/icc.js]: I wonder if we should just set style.top = StatusBar.height ?

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#bbeUNTwGRqGC_EcuMb_m5g

The pull request failed to pass integration tests. It could not be landed, please try again.

File: apps/system/js/icc.js (hunk 1 above)
Comment (albertopq) [inline apps/system/js/icc.js]: makes sense

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=07cf736c7abcee7325ab721def69f314791d5139

**Pull request merged**